File: add_simple/php/add.php
<?php

$body = file_get_contents('php://input');
$json = json_decode($body);
$json = $json->operands;

$r["result"] = 0;
foreach ($json as &$operand)
    $r["result"] += $operand;

header("Content-Type: application/json; charset=utf-8");
echo json_encode($r, JSON_PRETTY_PRINT | JSON_UNESCAPED_SLASHES);

?>

File: add_simple/python/add.py
import json

def application(environ, start_response):
    request_body_size = int(environ.get('CONTENT_LENGTH', 0))
    request_body = environ['wsgi.input'].read(request_body_size)
    request_json = json.loads(request_body)

    response = { "result": 0 }
    for value in request_json['operands']:
        response['result'] += value
    
    start_response('200 OK', [('Content-Type', 'application/json')])
    yield str.encode(json.dumps(response))
